docs/sdk/sfa/sfa-android/usage.mdx

```diff
@@ -43,12 +43,12 @@ Please checkout the [SessionData](#response) for more details.
 
 <TabItem value="table">
 
-| Parameter               | Description                                                                                                       |
-| ----------------------- | ----------------------------------------------------------------------------------------------------------------- |
-| `verifier`              | The verifier obtained from the Web3Auth Dashboard. It's a mandatory field and takes `String` as a value.          |
-| `verifierId`            | The verifierId used for the verification. It takes `String` as a value.                                           |
-| `idToken`               | The idToken of the user obtained from the login provider. It takes `String` as a value.                           |
-| `subVerifierInfoArray?` | Sub verifier info. Usually used during the aggregate verifier. It takes `Array<TorusSubVerifierInfo>` as a value. |
+| Parameter               | Description                                                                                                                                                                                                                   |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `verifier`              | The `verifier` parameter takes the name of the custom verifier from the Web3Auth Dashboard. This is a required field that must be a `String`. If you're using an aggregate verifier, make sure to pass the sub-verifier name. |
```

Comment:
`make sure to pass the sub-verifier name.` this line is incorrect.

They still pass the name of the aggregate verifier here. and in `subVerifierInfoArray` they pass in the sub verifier details.

Reply:
ugh, my bad. Nice catch, this is the behaviour of the Flutter SDK.